### ISSUE TITLE
invalid json

### ISSUE THREAD

**Issue by sjurco** [jwt-scala/jwt-scala#14]
in JwtUtils.scala these lines create invalid json once the value (or its toString representation) contains quotation marks "

``` scala
      case value: String => "\"" + value + "\""
      case value: Any => "\"" + value.toString + "\""
      case (key, value: String) => "\"" + key + "\":\"" + value + "\""
      case (key, value: Any) => "\"" + key + "\":\"" + value.toString + "\""
```

and the same applies to all

``` scala
"\"" + key + "\":\""
```

**Comment by pauldijou** [comment]
Good catch. On it.

**Comment by pauldijou** [comment]
Should be fixed in 0.5.1, which will land in Maven soon. Could you give it a try and tell me if it fixes your issue? Thanks.
